app/lucy/src/app/models/observation.ts

```diff
@@ -1,6 +1,7 @@
 import { MechanicalTreatment } from './MechanicalTreatment';
 
 export interface Observation {
+  updatedAt: any;
```

Comment:
Let create a utility method like that
const createEmptyObj = (obj: any, props: string[]) => {
    for (const k of props) {
           Object.defineProperty(obj, k, {
                 value: undefined,
                  writable: true
            });
     }
};